packages/react-instantsearch-core/src/connectors/connectNumericMenu.js

```diff
@@ -157,7 +156,7 @@ export default createConnector({
     if (!items.some(item => item.value === '')) {
       items.push({
         value: '',
-        isRefined: isEmpty(refinedItem),
+        isRefined: refinedItem === undefined,
```

Comment:
```suggestion
        isRefined: !refinedItem,
```

Reply:
Might be a bit much of a micro-optimization, but at least there’s less branches that wag